arc42/src/07_deployment_view.adoc

```diff
@@ -93,8 +93,20 @@ a| Set to the format of the QWAC certificate in the `tpp-qwac-certificate` heade
 | SPRING_DATASOURCE_URL                          | JDBC URL                            | _(not set)_
 | SPRING_DATASOURCE_USERNAME                     | DB username                         | _(not set)_
 | SPRING_JPA_PROPERTIES_HIBERNATE_DEFAULT_SCHEMA
-| DB schema name (depends on used DB and configuration)
-| _(not set)_
+| DB schema name (depends on used DB and configuration)                                | _(not set)_
+| SANDBOX_FEATURE_UI_CERTPAGE_ENABLED            | Enables/Disables the certificate page in the ui
+
+Possible values are: _true_ or _false_            | _true_
+| SANDBOX_UI_CONTACTMAILTO                       | Contact email address
+
+ Format: _mailto:YOUR@MAILADRESS.DE_ | psd2@adorsys.de
+| SANDBOX_UI_LOGO                                | URL to customized ui logo           | Points to the default psd2-accelerator logo embedded in the application
```

Comment:
The logo in the browser tab is not exchanged. Would be good if logo and tab logo are identical.

Reply:
Good point. We haven't talked about the favicon. I'll create a new task for that.